### PR TITLE
Bangladesh OCDID Update

### PR DESCRIPTION
**Pull request by NaviThotlu**
Hi @Zohour,

1. Added Bangladesh new constituencies in country-bd.csv
2. Added Bangladesh new constituencies in Jatiya_Sangsad_electorates.csv
3. Added README file.

### CODE REVIEW

**Comment by asmitajha91** [comment]
@azuser Hi Zohour, could you please help review this PR.

File: identifiers/country-bd/Jatiya_Sangsad_electorates.csv
Comment (HKSenior) [inline identifiers/country-bd/Jatiya_Sangsad_electorates.csv:1]: According to this [wiki](https://en.wikipedia.org/wiki/List_of_constituencies_of_the_Jatiya_Sangsad) Jatiya Sangsad has 8 divisions. The following districts aren't represented within the additions, please add them:
- Barishal of the Barisal district, 
- Jhalokati of the Barisal district
- Moulivibazar of the Sylhet district

File: identifiers/country-bd/README [hunk 1]
```diff
@@ -0,0 +1,9 @@
+File contents in `country-bd` directory:
+
+* Jatiya_Sangsad_electorates.csv:contains the list of electoral districts for the House of the Nation (Jatiya Sangsad) of Bangladesh
+
```
Comment (HKSenior) [inline identifiers/country-bd/README:4]: nit: remove new line from between list elements.

File: identifiers/country-bd/README [hunk 2]
```diff
@@ -0,0 +1,9 @@
+File contents in `country-bd` directory:
+
+* Jatiya_Sangsad_electorates.csv:contains the list of electoral districts for the House of the Nation (Jatiya Sangsad) of Bangladesh
```
Comment (HKSenior) [inline identifiers/country-bd/README:3]: nit: add a space after the colon.

```suggestion
* Jatiya_Sangsad_electorates.csv: contains the list of electoral districts for the House of the Nation (Jatiya Sangsad) of Bangladesh
```

**Review by HKSenior — CHANGES_REQUESTED** [review]

**Comment by NaviThotlu** [comment]
Hi @HKSenior,

1. We have updated Barisal to Barishal, but for Jhalokati and Moulivibazar we have checked those have names as Jhalokathi and Maulvibazar in the Bangladesh parliament website. Please refer to screen shot below.
image001

2.  we have updated README file with the changes mentioned.

File: identifiers/country-bd/Jatiya_Sangsad_electorates.csv [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+ocd-division/country:bd/ed:chittagong_11,Chittagong-11
+ocd-division/country:bd/ed:chittagong_12,Chittagong-12
+ocd-division/country:bd/ed:chittagong_13,Chittagong-13
+ocd-division/country:bd/ed:chittagong_14,Chittagong-14
+ocd-division/country:bd/ed:chittagong_15,Chittagong-15
+ocd-division/country:bd/ed:chittagong_16,Chittagong-16
+ocd-division/country:bd/ed:rangamati,Rangamati
+ocd-division/country:bd/ed:bandarban,Bandarban
+ocd-division/country:bd/ed:khagrachari,Khagrachari
+ocd-division/country:bd/ed:coxs_bazar_1,Cox s Bazar-1
```
Comment (HKSenior) [inline identifiers/country-bd/Jatiya_Sangsad_electorates.csv:263]: It's okay to have an apostrophe within the name column. Please add here and the following lines.

```suggestion
ocd-division/country:bd/ed:coxs_bazar_1,Cox's Bazar-1
```

Screenshot from website:
Screenshot 2023-10-25 at 8 10 20 AM

**Review by HKSenior — CHANGES_REQUESTED** [review]
Thanks for the clarification on the previous divisions.

**Comment by NaviThotlu** [comment]
Hi @HKSenior ,

we have added apostrophe in Cox's Bazar.

**Review by jloutsenhizer — APPROVED** [review]
Did a quick spot check, everything looks good to me.

**Review by jpmckinney — APPROVED** [review]
Looks good. Makes sense to omit the division and district from the OCD-ID, and just keep the constituency. (I quickly compared to https://en.wikipedia.org/wiki/List_of_constituencies_of_the_Jatiya_Sangsad)

**Pull request merged**